uncaught.js

```diff
@@ -9,18 +11,15 @@ var jsonStringify = require('json-stringify-safe');
 
 var tryCatch = require('./lib/try-catch-it.js');
 var errors = require('./errors.js');
+var structures = require('./structures.js');
+var CONSTANTS = require('./constants.js');
 
-var LOGGER_TIMEOUT = 30 * 1000;
-var SHUTDOWN_TIMEOUT = 30 * 1000;
-var PRE_LOGGING_ERROR_STATE = 'pre.logging.error';
-var LOGGING_ERROR_STATE = 'logging.error';
-var PRE_GRACEFUL_SHUTDOWN_STATE = 'pre.graceful.shutdown';
-var GRACEFUL_SHUTDOWN_STATE = 'graceful.shutdown';
-var POST_GRACEFUL_SHUTDOWN_STATE = 'post.graceful.shutdown';
+var ALL_STATES = [];
```

Comment:
how necessary is ALL_STATES? Seems everything is captured in the state machine and that the state machine is enough to store in `UncaughtExceptionStruct` prior to process.abort()